.github/workflows/prd-canary-watch.yml

```
@@ -66,24 +63,78 @@ jobs:
             echo "skip=false" >> "$GITHUB_OUTPUT"
           fi
 
-      - name: Fetch current + history rates (stub)
+      - name: Fetch current + history rates from Sentry Stats v2
         if: steps.recent.outputs.skip != 'true'
         id: rates
         env:
           TOKEN:   ${{ secrets.sentry-token || secrets.SENTRY_TOKEN }}
```

actionlint (1.7.12)

[error] property "sentry_token" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; github_token: string; sentry-token: string}

(expression)